packages/cozy-client/src/registry.js

```diff
@@ -72,13 +72,14 @@ class Registry {
    * @param  {string} params - Fetching parameters
    * @param  {string} params.type - "webapp" or "konnector"
    * @param  {string} params.channel - "dev"/"beta"/"stable"
+   * @param  {string} params.limit - maximum number of fetched apps - defaults to 200
    *
    * @returns {Array<RegistryApp>}
    */
   async fetchApps(params) {
-    const { channel, type } = params
+    const { channel, type, limit = 200 } = params
```

Comment:
nit: Since we fetch the first 300 for the store. Should we upgrade our default to 300 ?

Reply:
I prefer to have a lower default value to not forget to implement the pagination :)